### PR TITLE
Trinits auth

### PR DESCRIPTION
**Pull request by venky-1826**
Add Trinits logging  forms

### CODE REVIEW

File: ui/src/App.js [hunk 1]
```diff
@@ -30,6 +33,16 @@ function App() {
             <Route path="/report">
               <Report />
             </Route>
+            <Route path="/Forgatpasswordpage">
```
Comment (TriNitSTechnologies) [inline ui/src/App.js:36]: Change the path to '/forgot-password'

Comment (venky-1826) [reply ui/src/App.js]: changes done

File: ui/src/Components/LoginPage/ForgotPassword.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+    import { ErrorMessage, Field, Form, Formik } from "formik";
+import { Link } from "react-router-dom";
+import * as Yup from "yup";
+
+export default function Forgatpasswordpage() {
```
[the remark this answers is not in the document]
Comment (venky-1826) [reply ui/src/Components/LoginPage/ForgotPassword.js:5]: changes done

File: ui/src/Components/Header/Header.js [hunk 3]
```diff
@@ -24,7 +25,8 @@ function Header() {
           <div className="m-3 ms-2">
             <img src={photo} alt="circle" className="rounded-circle round" />
           </div>
-          <div className="fw-bold m-4 ms-1">Trinits Technologies</div>
+          <div className="fw-bold m-4 ms-1"> Trinits Technologies</div>
+         <Link to='/Loginpage'><button>login</button></Link>
```
[the remark this answers is not in the document]
Comment (venky-1826) [reply ui/src/Components/Header/Header.js:29]: changes done

File: ui/src/App.js [hunk 4]
```diff
@@ -30,6 +33,16 @@ function App() {
...
+              <Forgatpasswordpage />
+            </Route>
+            <Route path="/Register">
```
[the remark this answers is not in the document]
Comment (venky-1826) [reply ui/src/App.js:39]: changes done

File: ui/src/App.css [hunk 5]
```diff
@@ -16,6 +15,58 @@
   background-color: aliceblue;
   height: 13vh;
 }
+.appcss{
```
[the remark this answers is not in the document]
Comment (venky-1826) [reply ui/src/App.css]: changes done

File: ui/src/Components/LoginPage/Login.js [hunk 6]
```diff
@@ -0,0 +1,92 @@
+import axios from "axios";
+import { ErrorMessage, Field, Form, Formik } from "formik";
+import { FaRegEnvelope } from "react-icons/fa";
+import { FcGoogle } from "react-icons/fc";
+import { Link } from "react-router-dom";
+import * as Yup from "yup";
+
+export default function Loginpage() {
```
[the remark this answers is not in the document]
Comment (venky-1826) [reply ui/src/Components/LoginPage/Login.js]: done

File: ui/src/Components/LoginPage/ForgotPassword.js [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+          conformationemail: "",
+        
+        }}
+        validationSchema={Yup.object({
+          
+          conformationemail: Yup.string().required("Enter your email").trim(),
+         
+        })}
+        onSubmit={(values, { setSubmitting }) => {
+          setTimeout(() => {
```
[the remark this answers is not in the document]
Comment (venky-1826) [reply ui/src/Components/LoginPage/ForgotPassword.js]: setTimeout removed

File: ui/src/Components/LoginPage/ForgotPassword.js [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+  return (
+    <>
+      <Formik
+        initialValues={{
+          conformationemail: "",
+        
+        }}
+        validationSchema={Yup.object({
+          
+          conformationemail: Yup.string().required("Enter your email").trim(),
```
[the remark this answers is not in the document]
Comment (venky-1826) [reply ui/src/Components/LoginPage/ForgotPassword.js]: changed

**Review by TriNitSTechnologies — APPROVED** [review]
changes looks good to me

**Pull request merged**